tee-worker/litentry/pallets/account-linker/src/lib.rs

```diff
@@ -50,7 +50,7 @@ pub mod pallet {
 
 	use weights::WeightInfo;
 	pub const EXPIRING_BLOCK_NUMBER_MAX: u32 = 10 * 60 * 24 * 30; // 30 days for 6s per block
-	pub const MAX_ETH_LINKS: usize = 3;
+pub const MAX_ETH_LINKS: usize = 3;
```

Comment:
Strange that this indention doesn't get formatted

Reply:
Looks strange. Cargo fmt doesn't cover it.  Manually update the indention.

Reply:
guess becuase its not added to the workspace

the account-linker pallet will be deleted soon anyway